### PR TITLE
Changing app icon

### PR DESCRIPTION
**Pull request by inteljudajocu**
#### Description:

Changed the app icon from safe paths to COVID-DR and app name from safe paths to COVID-DR

#### Linked issues:

[Related ticket](https://trello.com/c/VZwTIEG5/110-cambiar-el-icono-de-safe-path-to-covid-rd)

#### Screenshots:

image

#### How to test:

Close the app and check the icon and name

### CODE REVIEW

**Review by adnanjt — COMMENTED** [review]
En IOS el nombre es el mismo actualmente safe paths parece que solo cambiaste el nombre en android @inteljudajocu

File: ios/COVIDSafePaths/Images.xcassets/AppIcon.appiconset/Contents.json [hunk 1]
```diff
@@ -1,122 +1,122 @@
...
     }
   ],
   "info" : {
-    "author" : "xcode",
-    "version" : 1
+    "version" : 1,
+    "author" : "xcode"
   }
-}
+}
```
Comment (D-Rosa99) [inline ios/COVIDSafePaths/Images.xcassets/AppIcon.appiconset/Contents.json]: you miss the new line at the end of the file

**Review by adnanjt — APPROVED** [review]
ios is now working looks good

**Review by D-Rosa99 — APPROVED** [review]
GOOD JOB

**Pull request merged**